spec/spec_helper.rb

```diff
@@ -11,6 +11,7 @@
 
 
 require 'rspec/rails'
+require 'devise'
```

Comment:
Does a devise require need to happen here? Wouldn't it happen with the environment load?